dm/tests/dmctl_command/run.sh

```diff
@@ -81,7 +81,7 @@ function run() {
 	cp $cur/conf/source1.yaml $WORK_DIR/source1.yaml
 	cp $cur/conf/source2.yaml $WORK_DIR/source2.yaml
 	sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker1/relay_log" $WORK_DIR/source1.yaml
-	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
+	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1 -w $WORKER1_NAME
```

Comment:
for checking the result of this feature, we can imitate this https://github.com/pingcap/tiflow/blob/d8dab8df9663b831d9ad6556ce48a9cfd0a9b770/dm/tests/ha_cases/run.sh#L345-L347